src/main/resources/view/DarkTheme.css

```diff
@@ -342,16 +342,20 @@
     -fx-background-radius: 0;
 }
 
-#tags {
+#dates {
     -fx-hgap: 7;
     -fx-vgap: 3;
 }
 
-#tags .label {
+.date {
     -fx-text-fill: white;
-    -fx-background-color: #3e7b91;
+    -fx-background-color: #28d837;
```

Comment:
Perhaps change colour to darker green, since text is white, will be easier to read